web/themes/joinup/prototype/sass/components/_listing.sass

```diff
@@ -171,7 +171,7 @@
 .listing__title--link a
   color: $color-orange
 
-.listing__card--title-pin
+.listing__card--corner-title
```

Comment:
I believe that could have been necessary when the contextual navigation menu has been on the right and the icon has been pushing the menu. But now the contextual navigation menu is on the left when the tile is either pinned or featured, so I don't think this is an actually useful tweak any more.

Reply:
Let's leave it as it is. This helper prevent overlapping of the pinned / featured icon and the title.